MCPTT/UI/CoversationView/View/ConversationViewController.swift

```diff
@@ -15,39 +15,38 @@ class ConversationViewController: UIViewController, UITextViewDelegate {
     @IBOutlet weak var pttToggleBtn: UIButton!
     @IBOutlet weak var messageText: UITextField!
     @IBOutlet weak var ChatCollectionView: UICollectionView!
+    @IBOutlet weak var PTTView: UIView!
```

Comment:
No variable name should start with a capital letter. Correct it at other places too.